bot/connector-web/src/main/kotlin/WebBuilders.kt

```diff
@@ -67,18 +75,87 @@ fun <T : Bus<T>> T.webButton(
         imageUrl
     )
 
+/**
+ * Creates a url button
+ */
+fun <T : Bus<T>> T.webUrlButton(title: CharSequence, url: String): Button =
+    UrlButton(
+        translate(title).toString(),
+        url
+    )
+
+/**
+ * Creates a postback button
+ */
+fun <T : Bus<T>> T.webPostbackButton(
+    title: CharSequence,
+    targetIntent: IntentAware? = null,
+    step: StoryStep<out StoryHandlerDefinition>? = null,
+    parameters: Parameters = Parameters()
+): Button =
+    PostbackButton(
+        translate(title).toString(),
+        targetIntent?.let { i -> SendChoice.encodeChoiceId(this, i, step, parameters.toMap()) }
+    )
+
+/**
+ * Creates a quickreply button
+ */
+fun <T : Bus<T>> T.webQuickReply(
+    title: CharSequence,
+    targetIntent: IntentAware? = null,
+    step: StoryStep<out StoryHandlerDefinition>? = null,
+    parameters: Parameters = Parameters()
+): Button =
+    QuickReply(
+        translate(title).toString(),
+        targetIntent?.let { i -> SendChoice.encodeChoiceId(this, i, step, parameters.toMap()) }
+    )
+
 /**
  * Creates a button from a text.
  */
 fun <T : Bus<T>> T.webTextButton(text: CharSequence): WebButton =
     WebButton(translate(text).toString())
 
 /**
- * Creates a [WebMessage] from a [MediaCard].
+ * Creates a [WebMessage] from a [WebCard].
  */
-fun <T : Bus<T>> T.webCard(card: MediaCard): WebMessage = WebMessage(card = card)
+
+fun <T : Bus<T>> T.webCard(card: WebCard): WebMessage = WebMessage(card = card)
+
+fun <T : Bus<T>> T.webCard(
+    title: CharSequence,
+    subTitle: CharSequence,
+    buttons: List<Button>
+): WebCard = WebCard(
+    title = title,
+    subTitle = subTitle,
+    buttons = buttons
+)
+
+fun <T : Bus<T>> T.webCardWithAttachement(
+    title: CharSequence,
```

Comment:
title and subtitle mandatory? Sometine we only want an image + 1 button ?

Reply:
Fixed